assets/js/base/components/reviews/review-sort-select/index.tsx

```diff
@@ -2,15 +2,25 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import PropTypes from 'prop-types';
 import SortSelect from '@woocommerce/base-components/sort-select';
+import { ChangeEventHandler } from 'react';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
-const ReviewSortSelect = ( { onChange, readOnly, value } ) => {
+interface ReviewSortSelectProps {
+	onChange: ChangeEventHandler;
+	readOnly: boolean;
+	value: string;
```

Comment:
Is there a reason why you defined the `value` type in this way? According to the old propTypes declaration, `value` can be only: 'most-recent', 'highest-rating', and 'lowest-rating'.

In TS, you can define it in this way 
```suggestion
	value: 'most-recent' | 'highest-rating' | 'lowest-rating'.;
```

Reply:
> Is there a reason why you defined the value type in this way? According to the old propTypes declaration, value can be only: 'most-recent', 'highest-rating', and 'lowest-rating'.

The reason I switched it to String is because I was getting the below error in assets/js/blocks/reviews/frontend-block.js file:

```
Type 'string' is not assignable to type '"most-recent" | "highest-rating" | "lowest-rating"'.ts(2322)

```

Maybe there is a better way to fix it?

Reply:
On `assets/js/blocks/reviews/frontend-block.js`, you could update the JSDoc in this way:
![image](https://user-images.githubusercontent.com/4463174/208407327-2e889711-1dd1-4c60-af10-7b824a32bea0.png)

Reply:
Thank you for the suggestion! I have updated the code. 👍